Source/WebCore/editing/Editor.h

```diff
@@ -607,6 +610,16 @@ class Editor : public CanMakeCheckedPtr<Editor> {
     bool isPastingFromMenuOrKeyBinding() const { return m_pastingFromMenuOrKeyBinding; }
     bool isCopyingFromMenuOrKeyBinding() const { return m_copyingFromMenuOrKeyBinding; }
 
+    Element* writingSuggestionsContainerElement();
+
+    WritingSuggestionData* writingSuggestionData() const { return m_writingSuggestionData.get(); }
+
+    bool isInsertingTextForWritingSuggestion() const { return m_isInsertingTextForWritingSuggestion; }
+
+    SingleThreadWeakPtr<RenderInline> writingSuggestionRenderer() const { return m_writingSuggestionRenderer; }
+
+    void setWritingSuggestionRenderer(RenderInline& renderer) { m_writingSuggestionRenderer = renderer; }
```

Comment:
I don’t think we need all these blank lines.

Not sure the return value from `writingSuggestionRenderer()` needs to be a `WeakPtr`. In our coding style I think it can be a raw pointer.